src/components/quizes/Kana.tsx

```diff
@@ -44,10 +48,16 @@ const Kana = () => {
   let chars: object[] = [];
 
   if (settingsObj) {
-    if (settingsObj.hiragana) chars = hiragana;
-    if (settingsObj.katakana) chars = chars.concat(katakana);
+    if (settingsObj.hiragana) {
+      chars = hiragana;
+      if (settingsObj.yoon) chars = chars.concat(hiraganaYoon)
+    }
+    if (settingsObj.katakana) {
+      chars = chars.concat(katakana);
+      if (settingsObj.yoon) chars = chars.concat(katakanaYoon)
+    }
```

Comment:
there's also the case where just `yoon` is selected

Reply:
Pushed a change to set chars to all `yoon` characters if it's the only thing selected.